index.js

```diff
@@ -208,6 +208,13 @@ module.exports = function leafletImage(map, callback) {
             });
         };
 
+        if (useAjax) {
+            getBase64(url, function(data) {
+                im.src = 'data: image/png;base64, ' + data;
+                im.onload();
+            });
+            return;
+        }
         im.src = url;
```

Comment:
This should be an else, right? Otherwise this both loads images via URL and loads them with ajax slightly after.